### DISCUSSION TITLE
Spare PCBs

### DISCUSSION THREAD

**Discussion by tonywright** [kachurovskiy/megaels#12] [General]
Hi, I am gathering all the parts to create the ELS to add to my lathe but the board is turning out expensive, at least the P&P is expensive from China. As you have to order 5 boards as a minimum I am guessing that a few people might have left over boards they don't need. Could I ask if someone is willing to sell me a board at half or quarter price that the manufacturer wants? I am sure p&p wouldn't really be that expensive. I live in the UK.

Thanks

Tony

**Comment by kachurovskiy** [comment]
Tony, I have 4 spare boards, happy to mail you 1 if you email me the Briefmarke from https://www.deutschepost.de/de.html that will point to your address (shipping 53g from Germany). You probably want "Großbrief bis 500 g". My address is m.kachurovskiy@gmail.com

**Reply by tonywright** [reply]
Hi Thanks, that will be great. I have tried to create postage via that site and it does not recognise my post code even though I have put "Great Britain" as my location, this is a compulsory field so I cannot progress.. The price is 3.7 euros, I can pay you via paypal if you have a link? Or any other way. Hopefully paypal will give you my address.

**Reply by kachurovskiy** [reply]
Tony, let's sort this out over the email - please contact me there. Thank you!

**Comment by oppozith** [comment]
[not an add] In case if Maxim have no objections.
Anyone in Ukraine you can buy boards for nanoELS and megaELS in Lviv for near 3 USD each
https://www.olx.ua/d/uk/obyavlenie/elektronna-gtara-dlya-tokarnogo-verstata-na-arduino-megaels-nanoels-IDPnBwt.html

**Reply by kachurovskiy** [reply]
Relevant ads are welcome, especially from Ukraine :)

**Reply by kachurovskiy** [reply]
BTW I'd be happy to also advertise your page selling kits (all the needed parts in 1 bag) or complete devices - assume you can ship to Europe and US - a lot of people ask for it as they can't or not willing to build ELS themselves.

**Comment by tonywright** [comment]
Oppozith, unfortunately, although I would like to support Ukraine in this difficult time, the postage from there to the UK looks even more expensive than it is from China :) Sorry.

**Reply by oppozith** [reply]
Nevermind. It will be helpful for local enthusiasts
